src/ui/dictionarybar.cc

```diff
@@ -19,12 +19,7 @@ DictionaryBar::DictionaryBar( QWidget * parent,
   editDictionaryCommand( _editDictionaryCommand ),
   maxDictionaryRefsInContextMenu( maxDictionaryRefsInContextMenu_ )
 {
-
-  auto iconWidth  = this->size().width();
-  auto iconHeight = this->size().height();
-
-  normalIconSize = { std::max( iconWidth, iconHeight ), std::max( iconWidth, iconHeight ) };
-
+  normalIconSize = { this->iconSize().height(), this->iconSize().height() };
```

Comment:
{height,height}?

Reply:
Yes, because iconSize's width can be anything, while height is bound by theme/platform...

Reply:
related:
https://github.com/xiaoyifang/goldendict-ng/pull/1750/commits/e8661877b49e5debf9dc3a3469fbb08480cf10e6

weird things happens when we consider using the width